### ISSUE TITLE
Add assets to support new design

### ISSUE THREAD

**Issue by pablofullana** [poanetwork/token-wizard#1083]
Child of #1082

**Comment by gabitoesmiapodo** [comment]
This could be tested, but it should be merged into some temporary branch, like "2.0-styles" where we continually merge the latest 2.0 updates, I think.

I don't think that extracting and adding all the new assets is the best thing to do at this step, so I'm doing that screen by screen, component by component, unless somebody needs some asset to do their job.

How to test:

- There should be some new folders and files, preparing the terrain for the new styles, check that those exist:

token-wizard/src/assets/stylesheets/v2.0 contains some new folders, files, and explanations.

token-wizard/src/assets/images/v2.0 has some folders and images for the new styles

This structure will allow us to have the old and new styles in parallel, minimizing the risk of losing something currently working. Once the work is finished, we'll delete the old files and that's it.

- It's possible to switch styles using: 

http://dappurl/?uiversion=2 (v2, currently ugly and broken)
http://dappurl/?uiversion=1 (old working version)
http://dappurl/  a mix of versions 1 and to, so as not to compensate whatever styles are not done yet.

This should work.

When we finish we will also delete the old styles.

**Comment by gabitoesmiapodo** [comment]
PR at https://github.com/poanetwork/token-wizard/pull/1119
